signac/common/configobj/__init__.py

```diff
@@ -1188,7 +1188,7 @@ def __init__(
             warnings.warn(
                 "Passing in an options dictionary to ConfigObj() is "
                 "deprecated. Use **options instead.",
-                DeprecationWarning,
+                FutureWarning,
```

Comment:
This file is vendored (copied from an upstream project) so we usually avoid changing it. We have control over how this object is created in the library and it's not a part of user-facing code, so I suggest reverting this change.
```suggestion
                DeprecationWarning,
```